index.html

```diff
@@ -444,23 +444,23 @@ <h3 class="modal-title">Modal title</h3>
                             <select data-hide-on-select="true" class="form-control" id="conclusion-language" name="conclusion-language" type="text"></select>
                         </fieldset>
                         <fieldset class="form-group">
-                            <label for="conclusion-homepage-url" data-toggle="tooltip" data-placement="top" title="Homepage URL for the conclusion.">Homepage URL</label>
+                            <label for="conclusion-homepage-url" data-toggle="tooltip" data-placement="top" title="Homepage URL for the conclusion. SCWB will present possible values from the scan.">Homepage URL   <i class="fa fa-check-circle" aria-hidden="true" title="scan-based"></i></label>
                             <select data-hide-on-select="true" class="form-control" id="conclusion-homepage-url" name="conclusion-homepage-url" type="text"></select>
                         </fieldset>
                         <fieldset class="form-group">
-                            <label for="conclusion-download-url" data-toggle="tooltip" data-placement="top" title="The download URL for obtaining the package.">Download URL</label>
+                            <label for="conclusion-download-url" data-toggle="tooltip" data-placement="top" title="The download URL for obtaining the package. SCWB will present possible values from the scan.">Download URL    <i class="fa fa-check-circle" aria-hidden="true" title="scan-based"></i></label>
                             <select data-hide-on-select="true" class="form-control" id="conclusion-download-url" name="conclusion-download-url" type="text"></select>
                         </fieldset>
                         <fieldset class="form-group">
-                            <label for="conclusion-license-url" data-toggle="tooltip" data-placement="top" title="Where available, the URL to the license text.">License URL</label>
+                            <label for="conclusion-license-url" data-toggle="tooltip" data-placement="top" title="Where available, the URL to the license text. SCWB will present possible values from the scan.">License URL    <i class="fa fa-check-circle" aria-hidden="true" title="scan-based"></i></label>
                             <select data-hide-on-select="true" class="form-control" id="conclusion-license-url" name="conclusion-license-url" type="text"></select>
                         </fieldset>
                         <fieldset class="form-group">
-                            <label for="conclusion-notice-url" data-toggle="tooltip" data-placement="top" title="A URL that contains notice text for a conclusion.">Notice URL</label>
+                            <label for="conclusion-notice-url" data-toggle="tooltip" data-placement="top" title="A URL that contains notice text for a conclusion. SCWB will present possible values from the scan.">Notice URL  <i class="fa fa-check-circle" aria-hidden="true" title="scan-based"></i></label>
```

Comment:
There seems to be some extraneous whitespace after the label's text?

Reply:
> There seems to be some extraneous whitespace after the label's text?

Okay ,so I'll reduce those extraneous spaces to one character space?

Reply:
We want to have no trailing white space at all.